instrumentation/struts-2.3/src/test/java/io/opentelemetry/javaagent/instrumentation/hypertrace/struts/Struts2Action.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright The Hypertrace Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.javaagent.instrumentation.hypertrace.struts;
+
+import com.opensymphony.xwork2.ActionSupport;
+import java.io.IOException;
+import org.apache.struts2.convention.annotation.ParentPackage;
+import org.apache.struts2.convention.annotation.Result;
+
+@Result(type = "json")
+@ParentPackage("json-default")
+public class Struts2Action extends ActionSupport {
+
+  private String jsonString;
+
+  public static String sample = "{'balance':1000.21,'is_vip':true,'num':100,'name':'foo'}";
+
+  public String body() throws IOException {
```

Comment:
Add `@Override` if the class overrides any method from the parent

Reply:
It does not override any method. This is our defined method which is an endpoint for `/body` calls.